docs/operator.md

```diff
@@ -220,14 +221,34 @@ below:
     name: cmk-install-dir
 ```
 
+### Multi socket support
+`CMK` is able to use multiple sockets. During cluster initialization, `init` module will distribute cores from all sockets
+across pools. To prevent a situation, where __dataplane__ pool or __controlplane__ pool are spawned only on a single socket
+operator is able to use one of two `mode` policies: `packed` and `spread`. This policies are configuring order of cores
+allocation:
+
+- __packed__ mode will put following cores order:
```

Comment:
`in the following order`. Note that this policy is not topology-aware, cores are assigned to pools "one after another"